static/admin/config.yml

```diff
@@ -119,23 +119,23 @@ collections:
     editor:
       preview: false
     fields:
-      - { label: "Title", name: "resume-page-title", widget: "string" }
+      - { label: "Title", name: "resume-page-title", widget: "string",pattern: ['.{15}', "Must have at least 15 characters"] }
       - label: "Work experience"
         name: "work-experience"
         widget: "list"
         fields:
-          - { label: "Job title", name: "job-title", widget: "string" }
-          - { label: "Company name", name: "company-name", widget: "string" }
-          - { label: "Job description", name: "job-description", widget: "string" }
+          - { label: "Job title", name: "job-title", widget: "string",pattern: ['.{3,50}', "Must have 3-50 characters"] }
+          - { label: "Company name", name: "company-name", widget: "string",pattern: ['.{3,20}', "Must have 3-20 characters"] }
+          - { label: "Job description", name: "job-description", widget: "string",pattern: ['.{80,4000}', "Must have 80-4000 characters"] }
           - { label: "Start date", name: "start-job-date", widget: "datetime" }
           - { label: "Finish date", name: "finish-job-date", widget: "datetime" }
       - label: "Education"
         name: "education"
         widget: "list"
         fields:
-          - { label: "School/University name", name: "school-name", widget: "string" }
-          - { label: "Course/Degree", name: "course", widget: "string" }
-          - { label: "Education description", name: "education-description", widget: "string" }
+          - { label: "School/University name", name: "school-name", widget: "string",pattern: ['.{9}', "Must have at least 9 characters"] }
+          - { label: "Course/Degree", name: "course", widget: "string",pattern: ['.{5,100}', "Must have 5-100 characters"] }
+          - { label: "Education description", name: "education-description", widget: "string",pattern: ['.{80,4000}', "Must have 80-4000 characters"] }
```

Comment:
4000 characters seems a bit long for an education description, have you checked whether it looks good?
If it looks good on desktop, tablet, and mobile then fine 😄